rhizome/common/bin/validate

```diff
@@ -0,0 +1,20 @@
+#!/bin/env ruby
```

Comment:
On my system `/bin` is a symlink to `usr/bin`. If that is true on the VM hosts, maybe 

```suggestion
#!/usr/bin/env ruby
```

Reply:
I did a quick grep and through the rhizome scripts, `/bin/env` is used 21 times and `/usr/bin/env` is used 2 times. For the consistency, let's leave it as is in this PR but discuss switching all?